tests/parsing/GlobbingTest.py

```diff
@@ -218,7 +219,8 @@ def setUp(self):
     def _test_glob(self, pattern, file_list):
         results = sorted([os.path.normcase(g) for g in glob(pattern)])
         file_list = sorted([os.path.normcase(f) for f in file_list])
-        self.assertEqual([i for i in results if "__pycache__" not in i],
+        self.assertEqual([i for i in results if re.search(r"(__pycache__|.pyc)",
```

Comment:
as this was the same before this won't belong into this commit. But indeed in this case we could change it, `i` normally means a number for me^^